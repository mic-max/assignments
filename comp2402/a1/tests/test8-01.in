help
helpful
helpfulness
helpfulnessity
h
help
HELP
HeLp
cheese
Che
chees
kill
kil
killabir
killa
bir
KillA
killa