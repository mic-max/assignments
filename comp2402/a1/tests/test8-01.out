help
helpful
helpfulness
helpfulnessity
HELP
HeLp
cheese
Che
kill
killabir
bir
KillA
